openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                    description: Details about the error
+                example:
+                  status: 400
+                  error: Could not parse transaction envelope.
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                example:
+                  status: 500
+                  error: An error occurred while processing this request.
+              example:
+                status: 500
+                error: An error occurred while processing this request.
+  /tss/transactions/build:
+    post:
+      tags:
+        - TSS
+      summary: Create transactions with wallet backend channel accounts as the source
+      description:
+        Create transactions with wallet backend channel accounts as the source, from a list of transactions where each transaction is represented as a list of operations that it contains.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                transactions:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      operations:
+                        type: array
+                        description: list of operations that make up the transaction
+                        items:
+                          type: string
+                          description: base64 operation xdr string
+                      timebounds:
+                        type: integer
+                        description: time in seconds after which this transaction will not be accepted by the network
```

Comment:
Is this the number of seconds _after the current time_ that the transaction expires, or the unix timestamp for when the transaction expires?

Reply:
Yeap, it is the number of seconds after the current time that the transaction expires. Changed this description to exactly that